modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/layout.js

```diff
@@ -436,28 +434,6 @@ AUI.add(
 
 			var eventHandles = [];
 
-			if (A.UA.ie || A.UA.edge) {
-				eventHandles.push(
-					BODY.delegate(
-						'mouseenter',
-						(event) => {
-							event.currentTarget.addClass('focus');
-						},
-						'.portlet'
-					)
-				);
-
-				eventHandles.push(
-					BODY.delegate(
-						'mouseleave',
-						(event) => {
-							event.currentTarget.removeClass('focus');
-						},
-						'.portlet'
-					)
-				);
-			}
```

Comment:
@kresimir-coko did we replace this somewhere? I would expect this to cause issues since we are removing a delegate that adds a class